sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -805,3 +816,37 @@ def test_apec_redshift_parameter_is_case_insensitive():
     mdl.REDshift = 0.2
     assert mdl.redshift.val == 0.2
     assert mdl.Redshift.val == 0.2
+
+
+@requires_xspec
+def get_xspec_models_with_exp_cache():
+    """Return the XSPEC models that use the modelCacher1d_exp cache.
+
+    Currently, that is: Models that inherit from XSAbsorptionModel.
+    """
+    models = get_xspec_models()
+    return [m for m in models if issubclass(m, XSAbsorptionModel)]
```

Comment:
Would it make sense to test the number of models that are found (more just as a regression test than anything else)?

Reply:
Sure, doesn't hurt.